app/main.py

```diff
@@ -14,4 +17,4 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    return phrase.lower().count(letter.lower())
```

Comment:
Remove redundant pycache files from the PR

Reply:
@Gr1zzZzzly You still have pycache files in the PR

Reply:
i think i removed all files now